home-page.html

```diff
@@ -204,48 +178,5 @@ <h2 class="footer-title">TEST</h2>
             </div>
             
         </div>
-
```

Comment:
Remove multiple spaces between lines through the file (can't place the comment in the right position)

Reply:
fixed